README.md

```diff
@@ -71,10 +71,9 @@ options for the offset in the file/text provided:
 ]
 ```
 
-To use the iOS SDK is to specified the `compilerargs` option. The value
-specified by `compilerargs` the `--` to must be prefixed:
+To use the iOS SDK, pass `-sdk` and `-target` arguments preceded by `--`:
 ```
-sourcekitten complete --text "import UIKit ; UIColor." --offset 22 --compilerargs -- '-target arm64-apple-ios9.0 -sdk /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS9.0.sdk'
+sourcekitten complete --text "import UIKit ; UIColor." --offset 22 -- '-target arm64-apple-ios9.0 -sdk /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS9.0.sdk'
```

Comment:
isn't the whole point of `--` that you _not_ pass in a single argument afterwards? In other words, shouldn't this example not have the enclosing quotes?